lib/IDE/Utils.cpp

```diff
@@ -849,9 +849,12 @@ bool swift::ide::isBeingCalled(ArrayRef<Expr *> ExprStack) {
     auto *AE = dyn_cast<ApplyExpr>(E);
     if (!AE || AE->isImplicit())
       continue;
-    if (auto *CRCE = dyn_cast<ConstructorRefCallExpr>(AE)) {
+    if (auto *CRCE = dyn_cast<ConstructorRefCallExpr>(AE->getFn())) {
       if (CRCE->getBase() == Target)
         return true;
+      if (auto *MCE = dyn_cast<ImplicitConversionExpr>(CRCE->getBase()))
+        if (MCE->getSubExpr() == Target)
+          return true;
```

Comment:
IMO this would be better written as:

```suggestion
      auto *Base = CRCE->getBase();
      while (auto *ICE = dyn_cast<ImplicitConversionExpr>(Base))
        Base = ICE->getSubExpr();
      if (Base == Target)
        return true;
```